2011-11-02  Samuel Thibault  <samuel.thibault@ens-lyon.org>

* bits/ioctl-types.h (_IOT_sgttyb): Set number of chars to 4.

---
 bits/ioctl-types.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/bits/ioctl-types.h
+++ b/bits/ioctl-types.h
@@ -64,7 +64,7 @@
 };
 
 #define	_IOT_sgttyb	/* Hurd ioctl type field.  */ \
-  _IOT (_IOTS (char), 6, _IOTS (short int), 1, 0, 0)
+  _IOT (_IOTS (char), 4, _IOTS (short int), 1, 0, 0)
 
 #if defined TIOCGWINSZ || defined TIOCSWINSZ
 /* Type of ARG for TIOCGWINSZ and TIOCSWINSZ requests.  */
